gui/tests/example.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { test, expect } from '@playwright/test';
+
+test('should contain the word "Welcome" on the homepage', async ({ page }) => {
```

Comment:
Is this the general style that Playwright test names follow? I would expect it would be more like `"homepage_welcome"` but I could be wrong.

Reply:
The test can be named whatever we want, so I can start using the certain naming convention we want to follow.